apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -298,9 +308,8 @@ impl Plugin for Telemetry {
                     .and_then(|h| h.to_str().ok())
                     .unwrap_or("");
                 let span = ::tracing::info_span!(ROUTER_SPAN_NAME,
-                    "http.method" = %router_request.method(),
                     "http.route" = %router_request.uri(),
-                    "http.flavor" = ?router_request.version(),
```

Comment:
oh do we remote http 1.1 now ?

Reply:
I removed that part because it's now part of the customized attributes we can add on spans, and it's called `network.protocol.version`